test/mergeSort_test.js

```diff
@@ -0,0 +1,45 @@
+import { expect } from 'chai'
+import { mergeSort, merge } from '../src/mergeSort'
+
+describe('mergeSort() and merge()', () => {
+
+  context('merge()', () => {
+
+    it('should be a function', () => {
+      expect(merge).to.be.a('function')
+    })
+
+    it('should add two sorted arrays together in ascending order', () => {
```

Comment:
Concatenate two arrays is the conventional way of expressing 'adding' arrays (to expand programmer vocabulary).